### PR TITLE
[ZEPPELIN-4941]. added support for personalized configuration of JDBCInterpreter database connection pool

### PR DESCRIPTION
**Pull request by xiejiajun**
### What is this PR for?
- Added support for personalized configuration of the JDBCInterpreter database connection pool. 
- This feature can be used to solve the runtime error caused by HiveServer2 Session timeout.
- This connection pool configuration Compatible with [the official configuration of dbcp2 ](http://commons.apache.org/proper/commons-dbcp/configuration.html)

### What type of PR is it?
[Feature]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-4941

### How should this be tested?
* manually tested
* Supported configuration items
    - validationQuery
    - testOnBorrow
    - testOnCreate
    - testOnReturn
    - testWhileIdle
    - timeBetweenEvictionRunsMillis
    - maxWaitMillis
    - maxIdle
    - minIdle
    - maxTotal

### Questions:
* Does the licenses files need update? NO
* Is there breaking changes for older versions? NO
* Does this needs documentation? NO

### CODE REVIEW

**Comment by zjffdu** [comment]
Thanks for the contribution @xiejiajun Could you update `jdbc.md` about this new feature ?

**Comment by xiejiajun** [comment]
> Thanks for the contribution @xiejiajun Could you update `jdbc.md` about this new feature ?

@zjffdu Ok, I have updated the jdbc.md document for this feature.

File: docs/interpreter/jdbc.md [hunk 1]
```diff
@@ -156,6 +156,25 @@ The last step is **Dependency Setting**. Since Zeppelin only includes `PostgreSQ
...
+| maxTotal                                                                                                                               | -1            | The maximum number of active connections that can be allocated from this pool at the same time, or negative for no limit.                                                                                                                                                             |
+| validationQuery                                                                                                                        | show database | The SQL query that will be used to validate connections from this pool before returning them to the caller. If specified, this query MUST be an SQL SELECT statement that returns at least one row. If not specified, connections will be validation by calling the isValid() method. |
+
+
```
Comment (zjffdu) [inline docs/interpreter/jdbc.md:177]: You can use html table to render this. Here's one example of spark interpreter.

https://github.com/apache/zeppelin/blob/master/docs/interpreter/spark.md

Comment (xiejiajun) [reply docs/interpreter/jdbc.md:222]: @zjffdu Okay, I have updated it with the html table syntax

Comment (xiejiajun) [reply docs/interpreter/jdbc.md:222]: > You can use html table to render this. Here's one example of spark interpreter.
> 
> https://github.com/apache/zeppelin/blob/master/docs/interpreter/spark.md

@zjffdu OK, I have updated the datasource pool docs in the  jdbc.md with the html table syntax.

**Pull request closed without merging**